### PR TITLE
Merge 38 c

### PR DESCRIPTION
**Pull request by japshvincent**

### CODE REVIEW

**Comment by kawesome** [comment]
Can one of the admins verify this patch?

File: src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java [hunk 1]
```diff
@@ -49,6 +49,7 @@
     private ReportRepository reportRepository;
     
     private AttendanceCourse existingCourse;
+    private AttendanceSection existingSection;
```
Comment (KSU-Github) [inline src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java]: MAJOR Remove this unused "existingSection" private field. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)

**Comment by KSU-Github** [comment]
SonarQube analysis reported 13 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 11 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ReportRepositoryITest.java#L64](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L64): Cast one of the operands of this division operation to a "float". [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS2184)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L57](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L57): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L58](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L58): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L59](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L59): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L60](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L60): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L61](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L61): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L62](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L62): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L63](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L63): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L65](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L65): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
1. ![MINOR][MINOR] [ReportRepositoryITest.java#L66](https://github.com/kstateome/lti-attendance/blob/6b694d5d99e53bebda5f207a84ad7dabc28abb9a/src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java#L66): Make this final field static too. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by japshvincent** [comment]
https://jenkins.ome.ksu.edu/job/Java/job/Test%20Deployers/job/LTI%20K-State%20Attendance%20(test1)/43/console

**Pull request merged**